### PR TITLE
fix: make code folding prettier for Init()

### PR DESCRIPTION
**Pull request by vito-laurenza-zocdoc**
@Zocdoc/platform

### CODE REVIEW

File: csharp/base.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ func Base(args []string) *BaseStatement {
 }
 
 func (self *BaseStatement) WriteCode(writer CodeWriter) {
-	writer.Write(" : base(")
+	writer.Write("base(")
```
Comment (vito-laurenza-zocdoc) [inline csharp/base.go:14]: this was bothering me so i cleaned it up too

File: csharp/property_test.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ MyType MyProperty
 	assertCode(t, property, expected)
 }
 
-func TestPropertyGetWithInitalizer(t *testing.T) {
+func TestPropertyGetWithInitializer(t *testing.T) {
```
Comment (vito-laurenza-zocdoc) [inline csharp/property_test.go:19]: got rid of spellcheck squiggles

File: csharp/method.go [hunk 3]
```diff
@@ -162,6 +162,7 @@ func (self *MethodDeclaration) WriteCode(writer CodeWriter) {
 		writer.Write(")")
 
 		if self.hasBase && self.base != nil {
+			writer.Write(" : ")
```
Comment (chris-smith-zocdoc) [inline csharp/method.go:165]: what is "base" on a method declaration?

Comment (vito-laurenza-zocdoc) [reply csharp/method.go:165]: it's used for Constructor Decls only, i.e. when `hasBase` is `true`.

File: csharp/base.go (hunk 1 above)
Comment (chris-smith-zocdoc) [reply csharp/base.go:14]: This was the spacing in the class inheritance? Or something else?

File: csharp/method.go (hunk 3 above)
Comment (vito-laurenza-zocdoc) [reply csharp/method.go:165]: i just moved the colon literal from one place to the other

File: csharp/base.go (hunk 1 above)
Comment (vito-laurenza-zocdoc) [reply csharp/base.go:14]: yes, sorry to mix this in this PR but i figured since i was touching this repo anyway...

**Review by chris-smith-zocdoc — APPROVED** [review]

**Pull request merged**